### PR TITLE
Fix improperly configured error showing

### LINKED ISSUE #82: Unable to get proper error
capture d ecran 2018-06-21 a 14 43 08

This is my conf :  

```
const configSvgSprite = {
    shape: {
        spacing: { // Add padding
            padding: 10
        }
    },
    mode: {
        css: { // Activate the «css» mode
            prefix: '.icon-',
            sprite: '../' + joli.build + '/assets/svg/',
            layout: 'vertical',
            bust: true, //Cache busting
            render: {
                scss: { // Activate Sass output
                    template: joli.src + '/styles/sprite-tmpl/',
                    dest: '../' + joli.src + '/styles/_sprite'
                },
            }
        },
    }
};

gulp.task('svgSprite', function () {
    return gulp.src([joli.src + '/svg/common/**/*.svg', joli.src + '/svg/' + joli.environment + '/**/*svg'])
        .pipe(svgSprite(configSvgSprite))
        .on('error', function(error){
            console.log(error);
        })
        .pipe(gulp.dest('out'));
});
```

### PR DESCRIPTION
**Pull request by Kreeg**
Fixes #82

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -13,6 +13,7 @@
 /** @typedef {import('stream').TransformOptions} TransformOptions */
 
 const { Transform } = require('stream');
+const util = require('util');
```
Comment (Kreeg) [inline index.js:16]: @XhmikosR I'm not sure how to proceed here
'node:utils' available after Node v14

Comment (XhmikosR) [reply index.js:16]: I suggest you just ignore the rule in package.json and we re-enable it after svg-sprite v3.0.0 where we drop support for Node.js < 14

**Review by XhmikosR — APPROVED** [review]

**Comment by XhmikosR** [comment]
I assume you tested it, @Kreeg, right? I don't really use gulp or grunt a lot nowadays :)

I can make a new release after this is merged.

**Comment by Kreeg** [comment]
изображение
изображение

I changed svg-sprite package's file to throw errors and it seems to do well
There is some refactoring ahead in the main package. Sometime we throw formatted error like
```
 error = util.format('SVGSpriter.add: "%s" is not the local part of "%s"', name, file);
```
Sometimes we rely on winston logger, like
```
spriter.error('Optimizing "%s" with SVGO failed with error "%s"', shape.name, error);
```
I think these should be unified

**Comment by XhmikosR** [comment]
Your call, but I agree we should be consistent. BTW, if we could replace winston with something smaller that has the same features we currently use that would make more sense.

**Comment by Kreeg** [comment]
@XhmikosR hello. I think we're ready for new release.

**Pull request merged**